src/mpyl/cli/projects.py

```diff
@@ -93,11 +93,25 @@ def shell_complete(self, ctx: click.Context, param, incomplete: str):
 
 @projects.command(name="show", help="Show details of a project")
 @click.argument("name", required=True, type=ProjectPath())
-@click.pass_obj
-def show_project(obj, name):
-    print_project(
-        obj.cli.repo, obj.cli.console, f"{name}/{Project.project_yaml_path()}"
-    )
+@click.pass_context
+def show_project(ctx, name):
+    obj = ctx.obj
+    project_path = f"{name}/{Project.project_yaml_path()}"
+    if not (obj.cli.repo.root_dir() / project_path).exists():
+        obj.cli.console.print(
+            Markdown(
+                f"Project `{name}` not found. 👉 Finding projects is much easier with [auto completion]"
+                f"(https://github.com/Vandebron/mpyl/blob/096df81404985d966b870e7d33697c0101563a88/"
+                f"README-usage.md#auto-completion) enabled."
```

Comment:
This could be the mpyl docs link:
https://vandebron.github.io/mpyl/mpyl.html#mpyl-cli